backend/src/main/resources/db/migration/V2__Create_Tables.sql

```diff
@@ -0,0 +1,28 @@
+BEGIN;
+
+CREATE TABLE tenant
+(
+    id              BIGINT AUTO_INCREMENT NOT NULL,
+    code            VARCHAR(255)          NULL,
```

Comment:
Code should be unique per tenant shouldn't it?

Reply:
Added new constraint